readme.md

```diff
@@ -106,6 +111,11 @@ flags: {
 		type: 'string',
 		alias: 'u',
 		default: 'rainbow'
+		isRequired: (flags, input) => {
+			if (flags.otherFlag) {
+				return true;
+			}
```

Comment:
This does not follow the docs as it doesn't return false for the negative case.

Reply:
Should `isRequired` throw if the return value is not a boolean?

Reply:
I've decided to implement a check for this that throws if the value returned is not a boolean.
I think it's a good idea, because just as I missed the negative condition in the example above, somebody might forget to return the proper values, and that might lead to some unexpected results.